website/django/data/tests.py

```diff
@@ -82,6 +93,186 @@ def test_autocomplete_bic(self):
         self.assertEqual(response.status_code, 200)
         self.assertJSONEqual(response.content, {"suggestions": expected_autocomplete_results})
 
+###################################################################################
+############################# NEW TESTS START #####################################
+    def test_ga4gh_variants_status_code(self):
+        request0 =  self.factory.post(
+            "/data/ga4gh/variants/search", json.dumps({"referenceName": "chr17", "variantSetId": "brca-hg37", "start": 51425158, "end": 515158029, "pageSize": 5 }), content_type="application/json")
+        response = variant_search(request0)
+        self.assertEqual(response.status_code, 200)
+
+    def test_ga4gh_translator(self):
+        var_resp = vrs.Variant()
+        resp = Variant.objects.values()[0]
+        genRefer = "hg37"
+        for j in resp:
+            if resp[j] == "-" or (resp[j] == ""):
+                continue
+            if j == "Genomic_Coordinate_"+genRefer:
+                refNme, strt, bases = resp[j].split(':')
```

Comment:
Sometimes with throwaway variables in a split you can just splat them to a _. Like `_, save = string.split(",")`